db/db_impl_readonly.cc

```diff
@@ -162,6 +162,7 @@ Status DB::OpenForReadOnly(
     for (auto cfd : *impl->versions_->GetColumnFamilySet()) {
       sv_context.NewSuperVersion();
       cfd->InstallSuperVersion(&sv_context, &impl->mutex_);
+      cfd->table_cache()->SetTablesAreImmortal();
```

Comment:
what if people are using read-only DB with finite table cache capacity, i.e., `max_open_files != -1`? Can tables be closed and memory unmapped?